staging/src/k8s.io/component-base/version/verflag/verflag.go

```diff
@@ -60,35 +60,29 @@ func (v *versionValue) Set(s string) error {
 	return err
 }
 
-func (v *versionValue) String() string {
+func (v *VersionValue) String() string {
 	if *v == VersionRaw {
 		return strRawVersion
 	}
 	return fmt.Sprintf("%v", bool(*v == VersionTrue))
 }
 
 // The type of the flag as required by the pflag.Value interface
-func (v *versionValue) Type() string {
+func (v *VersionValue) Type() string {
 	return "version"
 }
 
-func VersionVar(p *versionValue, name string, value versionValue, usage string) {
+func VersionVar(p *VersionValue, name string, value VersionValue, usage string) {
 	*p = value
 	flag.Var(p, name, usage)
 	// "--version" will be treated as "--version=true"
 	flag.Lookup(name).NoOptDefVal = "true"
 }
 
-func Version(name string, value versionValue, usage string) *versionValue {
-	p := new(versionValue)
-	VersionVar(p, name, value, usage)
-	return p
-}
-
 const versionFlagName = "version"
 
 var (
-	versionFlag = Version(versionFlagName, VersionFalse, "Print version information and quit")
+	versionFlag VersionValue
```

Comment:
I liked the old way better. Plus, that's how flags/pflags/Cobra usually does it.

Reply:
I see, I just liked the fewer LOC :smile: Changed it back how it was, now